datasource/ethereum/src/block_stream.rs

```diff
@@ -1096,5 +1225,188 @@ fn create_log_filter_from_subgraph(manifest: &SubgraphManifest) -> EthereumLogFi
                     (contract_addr, event_sig)
                 })
         })
-        .collect::<EthereumLogFilter>()
+        .collect::<EthereumLogFilter>();
+    match log_filter.contract_address_and_event_sig_pairs.len() {
```

Comment:
I generally prefer an empty line here and there to separate code blocks and longer assignments or call chains. E.g. I would add one above this line.